docs/release-notes/0.3/release-0.3.md

```diff
@@ -0,0 +1,65 @@
+# ML.NET 0.3 Release Notes
+
+Today we are releasing ML.NET 0.3. This release focuses on adding components to ML.NET from the internal codebase (such as Factorization Machines, LightGBM, Ensembles, and LightLDA), enabling export to the ONNX model format, and bug fixes.
+
+### Installation
+
+ML.NET supports Windows, MacOS, and Linux. See [supported OS versions of .NET
+Core
+2.0](https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0-supported-os.md)
+for more details.
+
+You can install ML.NET NuGet from the CLI using:
+```
+dotnet add package Microsoft.ML
+```
+
+From package manager:
+```
+Install-Package Microsoft.ML
+```
+
+### Release Notes
+
+Below are some of the highlights from this release.
+
+* Added Field-Aware Factorization Machines (FFM) as a learner for binary classification (#383)
+
+    * FFM is useful for various large sparse datasets, especially in areas such as recommendations and click prediction. It has been used to win various click prediction competitions such as the [Criteo Display Advertising Challenge on Kaggle](https://www.kaggle.com/c/criteo-display-ad-challenge). You can learn more about the winning solution [here](https://www.csie.ntu.edu.tw/~r01922136/kaggle-2014-criteo.pdf).
+    * FFM is a streaming learner so it does not require the entire dataset to fit in memory and can be used as an online learner.
```

Comment:
>online learner [](start = 114, length = 14)

repetitive.  Make crisper

Reply:
The referenced Kaggle competition seems a bit dated. Do we have any benchmarks showing its current utility vs. other methods?

Reply:
Addressed the repetition in next iteration. I am not aware of a more recent published benchmark.